### PR TITLE
fix: Wrong version in __init__

### PR DESCRIPTION
**Pull request by smacker**
Use only one source of truth for the version same as any other major
library does.

As examples of such libs: urllib3, requests, chardet, SQLAlchemy.

Signed-off-by: Maxim Sukharev <max@smacker.ru>

### CODE REVIEW

File: python/lookout/__init__.py [hunk 1]
```diff
@@ -2,4 +2,3 @@
 # DO NOT CHANGE OR ADD ANYTHING HERE
```
Comment (carlosms) [inline python/lookout/__init__.py]: I think the changes need to be done in https://github.com/src-d/lookout-sdk/blob/master/_tools/protogen_python.sh#L23

File: python/lookout/sdk/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+__version__ = "0.2.0"
```
Comment (vmarkovtsev) [inline python/lookout/sdk/__init__.py]: It is more common to specify this as a tuple:
```python
__version__ = 0, 2, 0
```

File: python/lookout/__init__.py (hunk 1 above)
Comment (smacker) [reply python/lookout/__init__.py]: ohhhh. thanks!
I don't think we should change anything besides generated code from proto when we run protogen.

File: python/lookout/sdk/__init__.py (hunk 2 above)
Comment (smacker) [reply python/lookout/sdk/__init__.py]: I also thought that but apparently, it's not true.

Here is example of 4 different VERY common libs all of them use string version:
https://github.com/urllib3/urllib3/blob/1.24.1/src/urllib3/__init__.py#L30
https://github.com/requests/requests/blob/master/requests/__version__.py#L8
https://github.com/chardet/chardet/blob/master/chardet/version.py#L8
https://github.com/zzzeek/sqlalchemy/blob/master/lib/sqlalchemy/__init__.py#L133

Though we can add `VERSION` variable also as [chardet](https://github.com/chardet/chardet/blob/master/chardet/version.py#L9)

File: python/lookout/__init__.py (hunk 1 above)
Comment (smacker) [reply python/lookout/__init__.py]: removed here: https://github.com/src-d/lookout-sdk/pull/55/commits/7ba612407f20c69611b59bc3a1830a1339317630

File: python/lookout/sdk/__init__.py (hunk 2 above)
Comment (vmarkovtsev) [reply python/lookout/sdk/__init__.py]: LOL you are right https://www.python.org/dev/peps/pep-0396
A good example of poor design... I have to live with it

Comment (carlosms) [reply python/lookout/sdk/__init__.py]: For reference, another example of string version can be found in the docs here: https://packaging.python.org/tutorials/packaging-projects/#creating-setup-py

Comment (smacker) [reply python/lookout/sdk/__init__.py]: Wow! The PEP is from 2011! But I have never heard of it!

**Comment by carlosms** [comment]
Oh wait, this change needs to be reflected here: https://github.com/src-d/lookout-sdk/blob/master/docs/CONTRIBUTING.md#release-process

**Comment by smacker** [comment]
🤦‍♂️ I learn something new about this repository every day.

Thanks, fixed!

**Pull request merged**